deployment/agent-deployment/ServiceGroupRoot/Scripts/pushAgentToAcr.sh

```diff
@@ -22,10 +22,10 @@ fi
 
 if [[ "$AGENT_IMAGE_FULL_PATH" == *"win-"* ]]; then
   echo "checking windows tags"
-  TAG_EXISTS=$(echo $MCR_TAG_RESULT | jq '.tags | contains(["'"win-$AGENT_RELEASE$AGENT_IMAGE_TAG_SUFFIX"'"])')
+  TAG_EXISTS=$(echo $MCR_TAG_RESULT | jq '.tags | contains(["'"win-$AGENT_IMAGE_TAG_SUFFIX"'"])')
 else
   echo "checking linux tags"
-  TAG_EXISTS=$(echo $MCR_TAG_RESULT | jq '.tags | contains(["'":$AGENT_RELEASE$AGENT_IMAGE_TAG_SUFFIX"'"])')
+  TAG_EXISTS=$(echo $MCR_TAG_RESULT | jq '.tags | contains(["'":IMAGE_TAG_SUFFIX"'"])')
```

Comment:
this seems to be incorrect. Should this be AGENT_IMAGE_TAG_SUFFIX?

Reply:
changed